doc/configuring_your_machine.md

```diff
@@ -82,46 +82,26 @@ The repo has a launch.json file that will launch the version of Python that is f
 
 2.  Invoke the CLI using:
 
-  #### OSX/Ubuntu (bash):
-  ```Shell
-  az
-  ```
-
-  #### Windows:
-  ```BatchFile
-  <clone root>\az.bat [commands]
+  #### Windows/OSX/Ubuntu:
   ```
-  which is equivalent to the following:
-  ```BatchFile
-  <clone root>\src\python -m azure.cli [commands]
+  az
   ```
 
 ## Running Tests:
 #### Command line
-  Provided your PYTHONPATH was set correctly, you can run the tests from your `<root clone>` directory.
+  Running the `dev_setup.py` file installed the test and style check scripts into the `tools` directory under your
+  `<root clone>` directory and added them into the current virtualenv.
```

Comment:
Should be `<clone root>` for consistency here.

Reply:
Good call
I've made another pr for this edit: https://github.com/Azure/azure-cli/pull/4845
Thanks!